actors/group5_geo.c

```diff
@@ -10,3 +10,15 @@
 #include "klepto/geo.inc.c"
 #include "eyerok/geo.inc.c"
 #include "pokey/geo.inc.c"
+
+#include "level_commands.h"
+
+const LevelScript script_func_global_6[] = {
+    LOAD_MODEL_FROM_GEO(MODEL_POKEY_HEAD,              pokey_head_geo),
```

Comment:
Revert the jump scripts back to their original files to be safe with fast64 exporting

Reply:
Fixed